phpcs.xml

```diff
@@ -342,17 +314,8 @@
   <rule ref="Squiz.WhiteSpace.LanguageConstructSpacing" />
   <rule ref="Squiz.WhiteSpace.SemicolonSpacing"/>
   <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
-  <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-    <properties>
-      <property name="ignoreNewlines" value="true"/>
-    </properties>
-  </rule>
```

Comment:
This checks operator spacing rules.

Here are few examples.

Correct: `$a = $b + $c`;
Incorrect: `$a=$b+$c`;

Correct
```php
  $abc = [
    "abc3" => "dssdsd3",
    "abc41" => [
      "dssdsd4"
    ],
  ];

  $abc = [
    "abc3" => "dssdsd3",
    "abc41" => 
    [
      "dssdsd4"
    ],
  ];
```

Incorrect

```php
$abc = [
    "abc3"=> "dssdsd3",
    "abc41" =>[
      "dssdsd4"
    ],
  ];
```